patches/net/minecraft/world/level/block/state/BlockBehaviour.java.patch

```diff
@@ -34,7 +34,7 @@
          return this.isRandomlyTicking;
      }
  
-+    @Deprecated //Forge: Use more sensitive version {@link IForgeBlockState#getSoundType(IWorldReader, BlockPos, Entity) }
++    @Deprecated //NeoForge: Use more sensitive version {@link IBlockStateExtension#getSoundType(LevelReader, BlockPos, Entity) }
```

Comment:
Would you mind changing this to `// Neo: `? That's what we use to reference our changes